examples/threathunter/threat_intelligence/feed_helper.py

```diff
@@ -9,7 +9,10 @@
 class FeedHelper():
     def __init__(self, start_date, size_of_request_in_minutes):
         self.size_of_request_in_minutes = size_of_request_in_minutes
-        self.start_date = datetime.strptime(start_date, "%Y-%m-%d %H:%M:%S").replace(tzinfo=timezone.utc)
+        if isinstance(start_date, datetime):
+            self.start_date = start_date.replace(tzinfo=timezone.utc)
+        else:
```

Comment:
If we want to be a little more defensive here you may want to check if start_date is a string then the else could be an error catch in case you don't receive a string. Is there a possibility you could receive an integer for an epoch time?